src/PureCloud.Client/Apis/PresenceApi.cs

```diff
@@ -22,6 +22,43 @@ public PresenceApi(IHttpClientFactory httpClientFactory, IOptions<PureCloudJsonS
         _options = options.Value.JsonSerializerOptions;
     }
 
+    /// <inheritdoc />
+    public async Task<bool> DeletePresenceDefinitionAsync(string definitionId, CancellationToken cancellationToken = default)
+    {
+        ArgumentException.ThrowIfNullOrEmpty(definitionId);
+
+        var uri = UriHelper.GetUri($"api/v2/presence/definitions/{Uri.EscapeDataString(definitionId)}", null);
+
+        var response = await _httpClient.DeleteAsync(uri, cancellationToken);
+
+        return response.IsSuccessStatusCode;
+    }
+
+    /// <inheritdoc />
+    public async Task<bool> DeletePresenceSourceAsync(string sourceId, CancellationToken cancellationToken = default)
+    {
+        ArgumentException.ThrowIfNullOrEmpty(sourceId);
+
+        var uri = UriHelper.GetUri($"api/v2/presence/sources/{Uri.EscapeDataString(sourceId)}", null);
```

Comment:
Don't use GetUri if there are no parameters. Be sure to apply this feedback everywhere